app/scripts/services/translation.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('translation', [])
+    .factory('Interpolate', [
```

Comment:
code style: lower case function name
otherwise it could be misleading that the factory returns a pseudo class